sydent/replication/peer.py

```diff
@@ -100,6 +102,9 @@ def pushUpdates(self, sgAssocs):
                                                                   '/_matrix/identity/replicate/v1/push',
                                                                   body)
 
+        # XXX: We'll also need to prune the deleted associations out of the
```

Comment:
I thought this was what https://github.com/matrix-org/sydent/pull/67/files#diff-8db67a35998f38ac70c2d2fa7d2d8200R90 was doing?

Reply:
Have tried to clarify in the comment